corehq/apps/users/views/utils.py

```diff
@@ -109,3 +110,23 @@ def log_commcare_user_locations_changes(request, user, old_location_id, old_assi
             fields_changed=fields_changed,
             change_messages=change_messages
         )
+
+
+def get_locations_with_cases(domain, location_ids):
```

Comment:
Correct me if I'm wrong about the way we assign cases to location-bound users, but as far as I have it there's two ways in which cases can owned:
1. Location owns the case (case `owner_id` is equal to `location_id`) <-- the way you catered for
2. Mobile worker owns the case (case `owner_id` is equal to the mobile user `user_id`)

If the second is true, I think it's even more important to include those cases (maybe only include those?), since the cases will move with the user, as opposed to the first where, even if the mobile user changes location, the case will still be owned by the location.

Reply:
It is true that mobile workers can own cases as well. I had a meeting with Neidy, and she said it could be fine to ignore the cases where the mobile worker owns the case. So essentially, the alert is only showing how many cases belong to each of the mobile worker's assigned locations.

I definitely do agree however, that we might be more interested in seeing which of the cases owned by the mobile worker would be moving with them. The only problem with getting cases owned by the mobile worker as well, is that how would one identify in which location those cases were created? The goal of the alert is to have a list of locations that have cases, and so if there is no way to tell which of the mobile worker's cases belongs where then it could be tricky to add those to the alert.

Reply:
> she said it could be fine to ignore the cases where the mobile worker owns the case

Oh, ok. Thinking of this now again, I remember when this ticket was refined we said the goal was to not leave cases as orphans, i.e. when a location owns cases and the last mobile worker is moved from that location, then the user should be aware that no CHW will attend to those anymore, whereas when the cases belongs to a CHW, that cases "moves" with the CHW.